doc/governance/index.rst

```diff
@@ -0,0 +1,116 @@
+Ceph Governance
+---------------
+
+Key principles
+==============
+
+  * Decision-making is consensus-driven by those who show up.
+  * Leadership roles are defined primary by responsibility, not prestige or seniority.
+    * It is normal and healthy for these roles to be passed on to others
+  * The role of developers is ultimately to serve the users.
+
+Bodies
+------
+
+Ceph Executive Council
+======================
+
+  * [3 or 5] people
+  * selected by steering committee during leadership summit
+    * candidates self-nominate or are nominated by other members
+    * discussion of how roles/responsibilities may be delegated
+    * propose 1 or more council sets
+    * ranked-choice vote by steering committee
+  * 1 year term (i.e., a release cycle) (or staggered longer terms?)
+
+Responsibilities
+^^^^^^^^^^^^^^^^
+ * arbiter in cases where decisions cannot be reached by consensus
+ * distribute key responsibilities amongst themselves or others
+ * ensure things get done
+
+Ceph Steering Committee
+=======================
+
+Membership
+^^^^^^^^^^
+
+ * senior developers, users, community members
+ * Target size: low 10s
+ * Existing CLT members are grandfathered in
+ * Members can be nominated and added/removed by existing members (2/3 majority)
+ * Term is normally 2 years, at which point they can be re-added or rotate out
+ * Membership reflected by a github group (should be on website,  docs, email list too)
+
+Responsibilities
+^^^^^^^^^^^^^^^^
+ * Select executive council
+ * Amend governance model
```

Comment:
```suggestion
 * Amend governance model. Changes to the governance model require a majority of the Ceph Steering Committee. 
```

Reply:
I'd lean towards the executive only being a tiebreaker in this scenario, not being required to agree.

On the 2nd question I think membership should be able to overlap, to allow the executive council members to also have a say in governance and future council members.